components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```diff
@@ -3464,7 +3592,7 @@ public static NativeObject jsFunction_refreshToken(Context cx, Scriptable thisOb
             String applicationName = (String) args[1];
             //String tokenType = (String) args[2];
             //Token type would be default with new scopes implementation introduced in 1.7.0
-            String tokenType = "default";
+            String requestedScopes = (String)args[8];
```

Comment:
I think requestedScopes should be a String array, because we should be able to accept more than one scopes from the UI isn't it?